frontend/src/app/Settings/Facility/FacilityForm.tsx

```diff
@@ -362,7 +363,7 @@ const FacilityForm: React.FC<Props> = (props) => {
                 className="margin-right-0 margin-top-05"
                 type="submit"
                 label="Save changes"
-                disabled={isSubmitting || !isDirty}
```

Comment:
this wasn't actually setting the save button to be disabled when we click the "Save changes" button. `isSubmitted` would do that but there's an issue if you back out of the address confirmation modal -- it won't reactivate the "Save changes" button so I opted to remove this.